resources/regions/IS_Queries_Mtb.multi.fasta

```diff
@@ -0,0 +1,14 @@
+>IS6110
+ATGTCCACCACCCGGGGGGAACCTCACACCTACTACGACGCCTGCGCCCGGCGCAGCCAGCCCTCCAAGC
+GCCAGGTCCGCGAAGAGGAGCTCAAAGTTGAGATCGCCCGTGTGCACCGCGAGAACTACGCGGTCTACGG
+AGCGCGCAAGGTATGGCTTCAGTGCCGCCGTGAGGGCGTCGAGGTGGCCCGCTGCACCGTCGAACGGCTG
+ATGGCCGAACTCGGCCTGCACGGAGCACGCAGAGGGAAGGTCAAGCGCACCACAATCGCCGACCCGCAAG
+CCGCTCGGCCCGATGACCTGGTTGGGCGACATTTCAGTCCTAATGCACCAAATATGTTGTGGGTAGCCGA
```

Comment:
This is the `multi-fasta` file which could be provided as a direct input to ISMAPPER 

<img width="1182" alt="image" src="https://github.com/user-attachments/assets/84a43fde-73a1-46b5-a279-0986d069c03a">

Reply:
Ah ok, ISmapper puts the results of each analysis into a separate folder with the name of sequence, could you then add this to the Nextflow dictionary or am I thinking about this wrong?

Reply:
Yes, we can just use the `glob` module in python script to collect those `txt` files